Makefile

```diff
@@ -89,36 +89,48 @@ ifeq ($(TARGET_ARCH),x64)
 	TARGET_ARCH_DEBIAN = amd64
 endif
 
-TEMPORARY_DIRECTORY = release/.tmp
 APPLICATION_NAME_LOWERCASE = $(shell echo $(APPLICATION_NAME) | tr A-Z a-z)
 APPLICATION_VERSION_DEBIAN = $(shell echo $(APPLICATION_VERSION) | tr "-" "~")
 
 # ---------------------------------------------------------------------
 # Rules
 # ---------------------------------------------------------------------
 
-release/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/node_modules: package.json npm-shrinkwrap.json
+release:
+	mkdir -p $@
+
+release/.tmp: | release
+	mkdir -p $@
+
+release/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies: | release
+	mkdir -p $@
+
+release/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/node_modules: package.json npm-shrinkwrap.json \
+	| release/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies
 	./scripts/unix/dependencies-npm.sh -p \
 		-r "$(TARGET_ARCH)" \
 		-v "$(ELECTRON_VERSION)" \
 		-x $(dir $@) \
```

Comment:
Obviously you need to be careful (and e.g. use the `word` macro) if there's more than one order-only pre-requisite.